src/NpmUtilities.js

```diff
@@ -17,8 +17,10 @@ function execInstall(directory, {
   // build command, arguments, and options
   const opts = NpmUtilities.getExecOpts(directory, registry);
   const args = ["install"];
-  let cmd = npmClient || "npm";
```

Comment:
This is perfectly acceptable, I'd prefer you didn't change code unrelated to the goal of the PR.

(this includes all the random deleted newlines, they're there for a reason)